build.sbt

```diff
@@ -113,6 +113,7 @@ lazy val `fs2-aws-kinesis` = (project in file("fs2-aws-kinesis"))
     Dependencies.Fs2Core,
     Dependencies.KinesisClient,
     Dependencies.KinesisProducer,
+    Dependencies.newTypes,
```

Comment:
are we moving away from refined types?

Reply:
i'm not sure if they are compatible with scala 3